packages/flutter/test/material/snack_bar_theme_test.dart

```diff
@@ -288,3 +298,12 @@ RawMaterialButton _getSnackBarButton(WidgetTester tester) {
     ).first,
   );
 }
+
+RenderParagraph _getSnackBarTextRenderObject(WidgetTester tester, String text) {
```

Comment:
Why not: `tester.renderObject<RenderParagraph>(...)`

Reply:
Wasn't aware of that, thanks!